balboa-client-jms/src/test/scala/com/socrata/balboa/impl/BufferTests.scala

```diff
@@ -121,6 +134,19 @@ with TestMessageQueueComponent with BeforeAndAfterEach {
       itemOk should be (true)
       testBuffer.bufferMap.size should be (3)
     }
+
+    "update buffer with multiple different metrics in different bufferItems" in new FullBufferSetup {
+      val entId = "marc"
+      val time3 = 1L + granularity/4
+      testBuffer.add(new BufferItem(entId, met6, time3))
+      val bufferItem = testBuffer.bufferMap.get(entId + ":" + testBuffer.timeBoundary(time3))
+      val itemOk = bufferItem match {
+        case None => false
+        case Some(b) => b.entityId == entId && equivalentMetrics(b.metrics, met7) && b.timestamp == testBuffer.timeBoundary(time3)
```

Comment:
Can we break up this boolean expression into pieces that can be better reasoned about?